embed-code-go/test/embedding_instruction_test/embedding_instruction_test.go

```diff
@@ -120,7 +120,8 @@ func (suite *EmbeddingInstructionTestSuite) TestReadFragmentDir() {
 	instruction, err := embedding_instruction.FromXML(xmlString, suite.config)
 	suite.Require().NoError(err, "There was unexpected error during the XML parsing.")
 
-	lines := instruction.Content()
+	lines, err := instruction.Content()
```

Comment:
Is it possible to avoid the duplicated code which you have introduced in this changeset?

```
	lines, err := instruction.Content()
	suite.Require().NoError(err, "There was unexpected error during receiving instruction content.")
```